### PR TITLE
feat(theme): manage tooltip with long text

### PR DESCRIPTION
**Pull request by charlesnguyen**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)
The text of the tooltip is cut when it is long

**What is the new behavior?**
the long text is broken and goes to the new line

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

File: packages/theme/example/index.html [hunk 1]
```diff
@@ -1470,13 +1470,17 @@ <h4 class="modal-title">Modal title</h4>
             </div>
             <h2>Tooltips</h2>
             <div id="tooltips-component" class="bs-component">
-              <button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="left" title="" data-original-title="Tooltip on left">Left</button>
+                <button type="button" class="btn btn-default" data-toggle="tooltip" data-placement="left" title="" data-original-title="Tooltip on left">Left</button>
```
Comment (talend-glorieux) [inline packages/theme/example/index.html]: Why the extra indentation ?

Comment (frassinier) [inline packages/theme/example/index.html]: I will fix it, probably auto-indent from his IDE.

File: packages/theme/src/theme/_tooltip.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.tooltip-inner {
+  word-break: break-all;
```
Comment (frassinier) [inline packages/theme/src/theme/_tooltip.scss]: I prefer `break-word` here.

Comment (talend-glorieux) [inline packages/theme/src/theme/_tooltip.scss]: `break-word`?

Comment (frassinier) [inline packages/theme/src/theme/_tooltip.scss]: Yes, but @charlesnguyen told me the problem with `_` in filename, so OK to merge with `break-all` option. LGTM

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://107.talend.surge.sh)

**Pull request merged**